prover/server/publish_prover.sh

```
@@ -0,0 +1,19 @@
+# 2.0.23
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)